recipe/bld.bat

```diff
@@ -126,3 +109,8 @@ pypy -m venv destination
 destination\Scripts\python -c "import sqlite3"
 if errorlevel 1 exit 1
 rmdir /q /s destination
+
+REM Copy required DLLs so pypy can be used without activating the environment
+copy /b %PREFIX%\Library\bin\libexpat.dll %PREFIX%
+copy /b %PREFIX%\Library\bin\libbz2.dll %PREFIX%
+copy /b %PREFIX%\Library\bin\ffi-8.dll %PREFIX%
```

Comment:
@isuruf this is the part that copies the 3 dlls in repsonse to #101

Reply:
We shouldn't copy them in this package. The copies will have precedence over the ones in `%PREFIX%\Library\bin` which get upgraded through their own conda packages.

Reply:
Otherwise, looks good to me.